src/go/rpk/pkg/utils/args.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package utils
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// Wrapping cobra's matchAll function and return an error including usage String.
+// MatchAll allows combining several PositionalArgs to work in concert.
+func MatchAll(checks ...cobra.PositionalArgs) cobra.PositionalArgs {
```

Comment:
Could you add some unit tests for this? Here's a couple tests that you could use as reference for the pattern we use the most: https://github.com/redpanda-data/redpanda/blob/dev/src/go/rpk/pkg/cli/cmd/wasm/generate_test.go

Reply:
Done, thanks for the reference.

https://github.com/r-vasquez/redpanda/blob/rpk-usage-for-invalid-args/src/go/rpk/pkg/cli/args_test.go